gee1k/uPic
# Add S3 private bucket access, and the selection between public and private button

## Summary of this pull request
This PR adds access to a private bucket via S3-Compatible. In the previous version, you will receive the 'public-read' error when you set up the bucket. I build a field to select between private and public-read. I can success to upload to a private bucket. 
## Does this solve an existing issue? If so, add a link to it
<img width="351" alt="image" src="https://github.com/gee1k/uPic/assets/83734772/b77b8d0e-aa42-421b-aa79-0fd1ec4ce5a5">

<img width="345" alt="image" src="https://github.com/gee1k/uPic/assets/83734772/207f1a97-e075-4cd1-8ff1-b4e869107683">

## Steps to test this feature
I create a private bucket on [Backblaze](https://www.backblaze.com/), which support the S3 API. But it should use the 'private' ACL to access the bucket. 

You can create a private bucket, and add the necessary information to the app.

## Screenshots
You can test the [link](https://img.jxpeng.dev/uPic/2023/07/G9GEoG.png).
## Additional info
